src/hct_mis_api/config/fragments/storages.py

```diff
@@ -1,24 +1,31 @@
 from hct_mis_api.config.env import env
 
-AZURE_ACCOUNT_NAME = env("STORAGE_AZURE_ACCOUNT_NAME", default="")
-AZURE_ACCOUNT_KEY = env("STORAGE_AZURE_ACCOUNT_KEY", default="")
+STORAGES = {
```

Comment:
This config seems weird
`_MEDIA` goes under key `staticfiles` but the coment indicates it's related to statics
`_DEFAULT` goes under key `default` but the comment indicates it being the media part